webapp-django/crashstats/crashstats/models.py

```diff
@@ -193,6 +194,7 @@ def inner(*args, **kwargs):
         self = args[0]
         url_or_implementation = args[1]
         if isinstance(url_or_implementation, basestring):
+            raise NotImplementedError(url_or_implementation)
             url = url_or_implementation
```

Comment:
Interesting mistake. I started with https://bugzilla.mozilla.org/show_bug.cgi?id=1357156 but found that the BugzillaBugInfo thing still uses `requests.get()` etc. Thanks to that `raise` this is how I discovered that. I guess I forgot to remove it afterwards. But will.